tests/importer/test_safari_dojson.py

```diff
@@ -462,6 +467,87 @@ def test_safari_audiobook(app):
         )
 
 
+def test_safari_audiobook2(app):
```

Comment:
Can we please add the leader fields accordingly in description for this test and the test above? It'll make the difference between the tests clearer!